src/SimpleAzure.Storage.HybridQueue/IHybridQueueExtensions.cs

```diff
@@ -0,0 +1,20 @@
+namespace WorldDomination.SimpleAzure.Storage.HybridQueues;

```

Comment:
WHOA! now this is kewl.

and lol .. i had this written up .. before i got to this..

![image](https://github.com/PureKrome/WorldDomination.SimpleAzure.Storage.HybridQueues/assets/899878/ac722ac1-7560-4e2e-a4ef-518f0c4692e5)

(i don't remember seeing this commit _by itself_ when i did my original 1x1 commit pass)

Reply:
The commit message explains it. Default interface members are rarely what anyone wants

Reply:
I don't believe I actually did a default interface method. I should be kicked off the internet. 🤮

TOTALLY accepting this.